test/ReactWrapper-spec.jsx

```diff
@@ -2073,17 +2073,37 @@ describeWithDOM('mount', () => {
   });
 
   describe('.hasClass(className)', () => {
-    it('should return whether or not node has a certain class', () => {
-      const wrapper = mount(
-        <div className="foo bar baz some-long-string FoOo" />,
-      );
+    context('When using a DOM component', () => {
```

Comment:
@ljharb are you OK with using `context` blocks here? I don't think we use them anywhere else, but it makes enough sense.